add info about incoming migration guide (#2431)
port-labs/port-docs: docs/build-your-software-catalog/sync-data-to-catalog/git/gitlab-v2/GitLab-v2.md

:::info GitLab v2 documentation
This page documents the latest GitLab integration, released in April 2025.  
For documentation of the previous integration, check out the [GitLab](/build-your-software-catalog/sync-data-to-catalog/git/gitlab/) page.  

> For users upgrading from v1, a migration guide will be released soon. Stay tuned for more updates!

Reviewer: ```suggestion
For users upgrading from the v1 integration, a migration guide will be available soon. Stay tuned!
```
